src/main/web/templates/error.ftl

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <meta charset="utf-8">
+    <title>Problem!</title>
+</head>
+<body>
+<p>We had an exception</p>
+<pre>${response.exception}</pre>
```

Comment:
In practice there are various reasons why an app would want to use this template. This is fine, it just assumes that if we end up here we have an exception.